### DISCUSSION TITLE
Support for shaking Wiimote

### DISCUSSION THREAD

**Discussion by geilo667** [ndeadly/MissionControl#168] [Ideas]
can you add support for shaking the wii controller?

**Comment by ndeadly** [comment]
It's in the works. For the regular wiimote it might not be particularly useful because it lacks the gyro that the switch is expecting. I don't know how much Switch games with motion rely on that component being present. The wiimotes with MotionPlus should be able to approximate proper motion controls, but I don't have access to one at the moment to test it out.
